src/lib/composite-layer.js

```diff
@@ -7,7 +7,15 @@ export default class CompositeLayer extends Layer {
 
   // initializeState is usually not needed for composite layers
   // Provide empty definition to disable check for missing definition
-  initializeState() {}
+  initializeState() {
+
```

Comment:
Nit: I could go either way on the braces but I don't think we need an empty line